src/main/webapp/app/exercises/shared/feedback/feedback.component.ts

```diff
@@ -64,7 +64,7 @@ export class FeedbackComponent implements OnInit {
      * the result.
      */
     @Input() showMissingAutomaticFeedbackInformation = false;
-    @Input() latestIndividualDueDate?: dayjs.Dayjs;
+    @Input() latestDueDate?: dayjs.Dayjs;
```

Comment:
Why did you rename this? I think it's more clearer with the 'individual'

Reply:
If you look at the rest call serverside you see that it is either the lastest individual due date or the latest exercise due date